### PR TITLE
[LTC] Implement Rapacious Guest

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[Rapacious Guest]]

**Comment by Susucre** [comment]
menace hint always got me.

File: Mage.Sets/src/mage/cards/r/RapaciousGuest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        // Menace
+        this.addAbility(new MenaceAbility(false));
+
+        // Whenever one or more creatures you control deal combat damage to a player, create a Food token.
+        this.addAbility(new DealCombatDamageControlledTriggeredAbility(
+                new CreateTokenEffect(new FoodToken())
+        ));
+
+        // Whenever you sacrifice a Food, put a +1/+1 counter on Rapacious Guest.
+        this.addAbility(new RapaciousGuestFoodTriggeredAbility());
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/r/RapaciousGuest.java:51]: Can we use `SacrificePermanentTriggeredAbility` here? (There are a few other cards that could be refactored to use it as well, search by `"Whenever you sacrifice a"`)

Comment (Susucre) [reply Mage.Sets/src/mage/cards/r/RapaciousGuest.java:51]: Oh missed that class. It is indeed exactly what was needed.

**Review by xenohedron — APPROVED** [review]

**Pull request merged**